x-pack/plugins/security_solution/server/endpoint/routes/policy/handlers.ts

```diff
@@ -63,3 +65,23 @@ export const getAgentPolicySummaryHandler = function (
     });
   };
 };
+
+export const getPolicyListHandler = function (
+  endpointAppContext: EndpointAppContext
+): RequestHandler<undefined, GetPackagePoliciesRequest['query'], undefined> {
+  return async (context, request, response) => {
+    const soClient = context.core.savedObjects.client;
+    const packagePolicyService = endpointAppContext.service.getPackagePolicyService();
+    try {
+      const listResponse = await packagePolicyService.list(soClient, request.query);
```

Comment:
See my prior comment around ensuring we are not creating a path to retrieve non-endpoint packages with this API. All calls to this API need to ensure they are being wrapped with a filter that looks only at endpoint integrations.  I'm ok if you want to do that here for now, but we may want to create a service module in our code base that can do that.